src/count_positives_sum_negatives.py

```diff
@@ -0,0 +1,15 @@
+"""A function to return an array of the count of positive elements and the sum of negative elements in an input array."""
+
+
+def count_positives_sum_negatives(arr):
+    """Return an array of the count of positive elements and the sum of negative elements in an input array.."""
+    sums_pos = 0
```

Comment:
This code can be done much more succinctly with a list comprehension, which is an awesome feature of Python. If you're not familiar with list comprehensions, look them up in the Python docs.